osidb/models/flaw/flaw.py

```diff
@@ -1063,9 +1079,9 @@ def _create_new_flaw():
                 return
 
             # we're handling an existing OSIDB-authored flaw -- update
-            if force_update or any(
-                getattr(old_flaw, field) != getattr(self, field)
-                for field in SYNC_REQUIRED_FIELDS
+            if force_update or (
+                diff is not None
+                and any(field in diff.keys() for field in SYNC_REQUIRED_FIELDS)
```

Comment:
The exact reason is that I wanted to make just minimal necessary changes to keep it working as it will probably all be rewritten by the migration to async sync managers :smile: Maybe I could remove/rework it too but it will be all reworked in a followup task.

Reply:
I see :smile: Then it makes sense, I was just confused as it seemed only halfway done :smile: